### DISCUSSION TITLE
Can implement the subsurface material like readme document

### DISCUSSION THREAD

**Discussion by fanbinnet** [google/filament#5899] [Q&A]
I want to implement the material like jade.But used the material sandbox's subsurface material, can't get the the effect I want.
The effect  I want is like below picture:
https://github.com/google/filament/blob/main/docs/images/samples/example_ssr.jpg
![jade](https://user-images.githubusercontent.com/3060688/183648534-efe69057-fa66-4a69-841a-bfec9c05d5c7.JPG)

Can someone give me the material parameters of the picture ?
Thanks very much!

**Comment by romainguy** [comment]
This example is implemented using refraction, not subsurface scattering.
